os/se/sss/security_hal_wrapper.c

```diff
@@ -51,7 +51,7 @@
 
 #define ISP_CHECKBUSY() while (isp_get_status()) {}
 
-#define HWRAP_LOG printf
+#define HWRAP_LOG vdbg
```

Comment:
Why do you use vdbg? This is not enough important to print with CONFIG_DEBUG_VERBOSE enabled.
Please use se-specific debug option.

Reply:
Thanks for the comment. Made a specific debug option and related code for security hal wrapper layer. Could you review the code if available.
Thanks again!